app/controllers/reports/housed_addresses_controller.rb

```diff
@@ -72,7 +77,8 @@ def column_types
         'Last Name' => { query: c_t[:last_name], display_type: :text },
         'Housing Address' => { query: md_b_t[:address], display_type: :address },
         _('Did you or this client use external software to help with housing?') => { query: md_b_t[:external_software_used], display_type: :boolean },
-        'Date Housed' => { query: md_b_t[:client_move_in_date], display_type: :text },
+        'Lease Start Date' => { query: md_b_t[:client_move_in_date], display_type: :text, display_transform: ->(dt) { dt.to_date } },
```

Comment:
does this blow up if `dt` is empty?  maybe?
```suggestion
        'Lease Start Date' => { query: md_b_t[:client_move_in_date], display_type: :text, display_transform: ->(dt) { dt&.to_date } },
```

Reply:
it does but `client_move_in_date` is always present https://github.com/greenriver/boston-cas/blob/4fdd885f542778ae22b49b61f6068b1e271eb6c3/app/controllers/reports/housed_addresses_controller.rb#L83

Reply:
but I can update it anyway

Reply:
thanks.  I thought that might be the case, but hadn't tracked it that far back.